yaml/virtual-copy/virtual-copy-base-vol-tc2.yml

```diff
@@ -20,24 +20,24 @@ metadata:
 provisioner: csi.hpe.com
 parameters:
   csi.storage.k8s.io/fstype: ext4
-  csi.storage.k8s.io/provisioner-secret-name: ci-primera3par-csp-secret
+  csi.storage.k8s.io/provisioner-secret-name: hpe3parprimera-secret-tc2
   csi.storage.k8s.io/provisioner-secret-namespace: kube-system
-  csi.storage.k8s.io/controller-publish-secret-name: ci-primera3par-csp-secret
+  csi.storage.k8s.io/controller-publish-secret-name: hpe3parprimera-secret-tc2
   csi.storage.k8s.io/controller-publish-secret-namespace: kube-system
-  csi.storage.k8s.io/node-stage-secret-name: ci-primera3par-csp-secret
+  csi.storage.k8s.io/node-stage-secret-name: hpe3parprimera-secret-tc2
   csi.storage.k8s.io/node-stage-secret-namespace: kube-system
-  csi.storage.k8s.io/node-publish-secret-name: ci-primera3par-csp-secret
+  csi.storage.k8s.io/node-publish-secret-name: hpe3parprimera-secret-tc2
   csi.storage.k8s.io/node-publish-secret-namespace: kube-system
   # Uncomment for k8s 1.14 for resize support
   #csi.storage.k8s.io/resizer-secret-name: nimble-secret
   #csi.storage.k8s.io/resizer-secret-namespace: kube-system
   # Uncomment for k8s 1.15 for resize support
   #csi.storage.k8s.io/controller-expand-secret-name: nimble-secret
   #csi.storage.k8s.io/controller-expand-secret-namespace: kube-
-  cpg: "CI_CPG"
+  cpg: "SSD_r6"
```

Comment:
The same I see with cpgs. These are the old file values. The CPG was changed to CI_CPG as this was common across arrays and in the default domain. This is ensure that the test runs for both primera and 3par

Reply:
Addressed the comment and made appropriate changes. Please review.